src/extension/_locales/en/messages.json

```
@@ -152,6 +149,9 @@
   "config_delete_all_confirm": {
     "message": "Do you really want to delete all settings?"
   },
+  "config_unsupported_legacy": {
+    "message": "This site is no longer supported"
```

Shouldn't we add something like:
```suggestion
    "message": "This site is no longer supported. Please migrate to Helix 3."
```

---

Ok, just saw https://github.com/adobe/helix-sidekick/pull/237#issuecomment-1019224622.
Probably makes more sense here.